package/PKGBUILD

```diff
@@ -3,7 +3,7 @@ targets=(
   "centos"
 )
 pkgname="carbonio-files-ce"
-pkgver="0.4.1"
+pkgver="0.4.1.prom"
 pkgrel="1"
```

Comment:
The `pkgver` attribute should remain 0.4.1. You should change the `pkgrel` attribute